pkg/monitoring/monitoring.go

```diff
@@ -71,6 +74,9 @@ var (
 	}
 )
 
+// prefix of monitoring catalogID
+var cattleMonitoringPrefixCatalogID = "catalog://?catalog=system-library&template=rancher-monitoring&version=%s"
```

Comment:
Nit: we have to put `system-library` to the constant. It is being hardcoded in several places.